log-ffi/src/lib.rs

```diff
@@ -0,0 +1,46 @@
+// Copyright 2025-Present Datadog, Inc. https://www.datadoghq.com/
```

Comment:
Do you think there is any value in adding something for log-ffi to [examples](https://github.com/DataDog/libdatadog/tree/64fd851081732a8849dcc17dc38298d5ef419f1f/examples/ffi)?

Reply:
good call, a dumb example would be good for people understand the usage.

Reply:
I didn't add its own example but updated the trace exporter which actually emits some logs
```
➜ ./examples/ffi/build/trace_exporter                                                                                               
ERROR :: Error deserializing trace from request body :: err: "Invalid format: Unable to read array len for trace count"
Operation failed with error: 18, reason: Invalid format: Unable to read array len for trace count
TRACE :: checkout waiting for idle connection: ("http", localhost:8126) :: 
TRACE :: Http::connect; scheme=Some("http"), host=Some("localhost"), port=Some(Port(8126)) :: 
TRACE :: checkout dropped for ("http", localhost:8126) :: 
ERROR :: Error while fetching /info :: err: client error (Connect)
```